src/python_interpreter/mod.rs

```diff
@@ -350,6 +352,10 @@ pub struct PythonInterpreter {
     /// When cross compile the target interpreter isn't runnable,
     /// and it's `executable` is empty
     pub runnable: bool,
+    /// Comes from `sys.platform.name`
+    pub implmentation_name: String,
+    /// Comes from sysconfig var `SOABI`
+    pub soabi: Option<String>,
```

Comment:
Ideally `implmentation_name` and `soabi` should be moved to `InterpreterConfig`, but that requires updating all of the bundled `sysconfig` files, we can do that later.